addons/mail/static/src/web/thread_service_patch.js

```diff
@@ -188,6 +189,15 @@ patch(ThreadService.prototype, "mail/web", {
             this.chatWindowService.notifyState(chatWindow);
             return;
         }
+        if (thread.model !== "discuss.channel") {
```

Comment:
Shouldn't it also open the form view even in mobile? (not done because of the above condition)